shell/clean.sh

```diff
@@ -19,9 +17,7 @@ for directory in packages/*; do
 
             # Remove node modules
             rm -fr node_modules;
-
-            # Remove jspm packages
-            rm -fr jspm_packages;
+            rm -f yarn.lock;
```

Comment:
I don't think we want to remove this, this would be like removing system.config.js. Don't we want dependencies of dependencies locked?